### DISCUSSION TITLE
Is it possible to put calendar in page. By default calendar show.

### DISCUSSION THREAD

**Discussion by anowarsadata** [dmuy/duDatepicker#20] [Q&A] [answered]
Is it possible to put calendar in page. By default calendar show.

**Comment by dmuy** [comment, marked as answer]
Hi @anowarsadata,

I'm sorry, this plugin is used as a picker only, so by default it is not shown and the user should click on an input. If you are looking for a calendar plugin (not a picker), you should check FullCalendar instead.
